### ISSUE TITLE
[BUG] bash_completion: -v switch is error, --version works correctly

### ISSUE THREAD

**Issue by arrowgent** [dosbox-staging/dosbox-staging#1472]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.78.0

### What Operating System are you using?

Linux x86_64

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

dosbox -v causes error
dosbox --version does not

see dosbox -h for example:
```
  -v, --version       Output version information and exit.

You can find full list of options in the man page: dosbox(1)
And in file: /usr/share/doc/dosbox-staging/README
```

https://github.com/dosbox-staging/dosbox-staging/issues/1471

### Steps to reproduce the behaviour.

Explain how to reproduce
1. ./dosbox -v
2. ./dosbox --version

### Provide a Log

```text
./dosbox-staging -v
Stack trace:
0       0x5570b9f5e35a ./dosbox-staging(+0x4135a) [0x5570b9f5e35a]
2021-12-28 11:46:18.297 | CHECK FAILED:  arg_it < argc  (2 < 2)  Missing verbosiy level after -v
Aborted
./dosbox-staging --version
date       time         | 
2021-12-28 11:46:21.040 | arguments: ./dosbox-staging --version
2021-12-28 11:46:21.040 | Current dir: /home/catbox/.local/games/dosbox-staging
2021-12-28 11:46:21.040 | stderr verbosity: 0
2021-12-28 11:46:21.040 | -----------------------------------
dosbox (dosbox-staging), version 0.78.0-2-g33cca879

Copyright (C) 2020-2021  The DOSBox Staging Team
License: GNU GPL-2.0-or-later <https://www.gnu.org/licenses/gpl-2.0.html>

This is free software, and you are welcome to change and redistribute it under
certain conditions; please read the COPYING file thoroughly before doing so.
There is NO WARRANTY, to the extent permitted by law.
2021-12-28 11:46:21.040 | atexit
```

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by arrowgent** [comment]
also verbosity typo

https://github.com/dosbox-staging/dosbox-staging/blob/ff280a88b7acaaf470e71059a733ce549b7638b3/src/libs/loguru/loguru.cpp#L501

**Comment by arrowgent** [comment]
missing flag/switch with --version include -v

https://github.com/dosbox-staging/dosbox-staging/blob/ff280a88b7acaaf470e71059a733ce549b7638b3/contrib/linux/bash-completion/dosbox#L33

**Comment by arrowgent** [comment]
-v conflict,

with loguru -v is verbosity level
`dosbox -v 0` is default examples `dosbox -v 1` or `dosbox -v 10` shows:
`2021-12-28 12:09:45.413 | stderr verbosity: 10`


remove the man info for `-v` or change verbosity to -log ?
making it `dosbox -log #` to modify verbosity level?

**Comment by kcgen** [comment]
Yes to all :+1: , thanks for delving into the new logger, @arrowgent !

Regarding disambiguating verbosity from version: going with the most common pattern sounds best. 
 - Personally, I'd be happy with `-v` setting (or boosting) verbosity level and supporting exact levels, like you mentioned.
 - For version: `--version` or `-V` seem common.

That said, if there's a "CLI best practices" guide out there or CLI for GNU programs - let's use that.

As for adjusting levels: right now, Loguru was added as identical API into the code base exactly like DOSBox upstream uses: `LOG_MSG(...)`.  We've also started using `LOG_INFO(...)` and `LOG_WARNING(...)`, as a beging.  But in general - we need more help in this area to add more logging and flagged with the appropriate level, just like you've indicated.

**Comment by arrowgent** [comment]
well that still leaves the issue of
-v giving an error and halting dosbox `Stack trace:`
when -h suggests it would report -v, --version

if `dosbox -v` is sent, then it should not fail to launch dosbox, instead it should interpret as `dosbox -v 0` or give meaningful error output that doesnt require seeking the source code for an answer

**Comment by Grounded0** [comment]
Fixed by #1478.
